CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+
+* Create a topic branch from the `master` branch. Like this: `git checkout -b add/background-clip-text`
+tip: use 'add', 'fix' or 'enhancement' to indicate on your type of contribution
+
+* To add support for a CSS feature you must call the convert function inside of `data/prefixes.js` and specify three parameters: data, options and a callback. Like this:
+  ```
+  f(require('caniuse-lite/data/features/background-clip-text'), browsers =>
+    prefix(['background-clip'], {
+      feature: 'background-clip-text',
+      browsers
+    })
+  )
+  ```
+* In order to add support for a new feature we need to add a hack to the `libs/hacks` folder using the name of the CSS feature as the filename. Tip: 'hack' is the actual support for the new feature.
+
+* Create a class that extends `Value`. In his new class change the prefix for `IE` to `-webkit-`. [See complete example](https://github.com/postcss/Autoprefixer/blob/73c7b6ab090a9a9a03869b3099096af00be7eb7d/lib/hacks/background-clip.js)
+
+* Load the new class in `lib/prefixes.js`. We refer to a new added class as a 'hack'. To load it we use `Declaration` module and call the `hack` method on the new class we created earlier. Like this:
+  ```
+  Declaration.hack(require('./hacks/background-clip'))
+  ```
+  [See detailed example](https://github.com/postcss/autoprefixer/blob/73c7b6ab090a9a9a03869b3099096af00be7eb7d/lib/prefixes.js)
+
+* Add unit tests to:
+  - `test/autoprefixer.test.js` to test that Autoprefixer works with Chrome and Edge browsers adds only -webkit- prefix. [See details here](https://github.com/postcss/autoprefixer/commit/73c7b6ab090a9a9a03869b3099096af00be7eb7d)
+  - `test/cases/background-clip.css` TODO: Explain how to add unit test to the CSS
+  - Run yarn test
+  - Push branch with updated commits to your copy of the fork
+  - Create and send PR to Autoprefixer
+
+
+## Changing an existing prefix
+TODO
+
+## Fixing Bugs
+1. Create
+2. Add test (same as above)
+3. You can send PR. The issue with tests if better than issue just with text
+4. But if you fix it, it will be always better
+5. yarn test
+6. Send PR
+
+## Filing issues
+Issues, questions, or comments about existing features should be filed as an issue. Support for new features also requires filing a new issue. You can file an issue [here](https://github.com/postcssAautoprefixer/issues).
+
+- If you have problems with the output CSS from Autoprefixer; you'll need to post:
+
+  1. Input CSS
+  2. Output CSS
+  3. Expected output CSS
+  4. Run `npm ls | grep autoprefixer` and check output
+  5. Browserslist config. If you use `browsers` option, create Browserslist config first.
+
+- If Autoprefixer throws error:
+
+  1. Input CSS
+  2. Run `npm ls | grep autoprefixer` and check output
+  3. Run `npm ls | grep postcss` and check output
+  4. Browserslist config
+  5. Error stacktrace (with example for users who don’t know what is stacktrace)
```

Comment:
`with example for users who don’t know what is stacktrace` :D

Reply:
Do you want to add any random error stack trace as an example? Or are we talking about how to format a line of code using code blocks to better read a given stack trace?

Reply:
Random error stack. I have few issue when users doesn’t know ”stacktrace” word

Reply:
Done on `7c851a7`. I think I understood what you wanted, let me know if you want to rephrase this.